Rakefile

```diff
@@ -119,3 +119,40 @@ def xcodebuild(command)
     sh "xcodebuild #{command}"
   end
 end
+
+def formatTool(command)
+  # As of Xcode 13.4 / Xcode 14 beta 4, including airbnb/swift as a dependency
+  # causes Xcode to spin indefinitely at 100% CPU (due to the remote binary dependencies
+  # used by that package). As a workaround, we can specifically add that dependency
+  # to our Package.swift file when linting / formatting and remove it afterwards.
+  packageDefinition = File.read('Package.swift')
+  packageDefinitionWithFormatDependency = packageDefinition +
+  <<~EOC
+  
+  #if swift(>=5.6)
+  // Add the Airbnb Swift formatting plugin if possible
+  package.dependencies.append(
+    .package(
+      url: "https://github.com/airbnb/swift",
+      .upToNextMajor(from: "1.0.1")))
```

Comment:
Oh I almost forgot, since we aren't checking in a `Package.resolved` for this we need to list a specific commit here

Reply:
```suggestion
      // Since we don't have a Package.resolved for this, we need to reference a specific commit
      // so changes to the style guide don't cause this repo's checks to start failing.
      .revision("cec29280c35dd6eccba415fa3bfc24c819eae887")))
```